zombies.js

```diff
@@ -78,8 +83,90 @@
  * @property {method} getPack              Returns private variable `pack`.
  * @property {method} getMaxHealth         Returns private variable `maxHealth`.
  */
-
-
+function Player(name, health, strength, speed) {
+  this.name = name;
+  this.health = health;
+  this.strength = strength;
+  this.speed = speed;
+  var pack = [];
+  var maxHealth = health;
+  this.isAlive = true;
+  this.equipped = false;
+
+  this.getMaxHealth = function() {
+    return maxHealth;
+  };
+  this.checkPack = function() {
+    return console.log(this.pack);
+  };
+  this.getPack = function() {
+    return pack;
+  };
+  this.takeItem = function(item) {
```

Comment:
All the methods below here should be on the prototype and not in the constructor function.